### ISSUE TITLE
Speed hacks

### ISSUE THREAD

**Issue by hazmanrussell** [fr1kin/ForgeHax#242]
Is there any way to change the speed at which you run with this?

**Comment by fr1kin** [comment]
Yeah pretty much. You can increase the speed by doing `boatfly speed <number>` in the console (open by pressing right-shift). Speed ranges from 0 to 10; anything above 10 will probably get you set back by vanilla but I am not entirely sure.
